docker-compose.ci.yml

```diff
@@ -10,7 +10,6 @@ services:
       DB_HOST: ${DB_HOST:-pg}
       DB_PORT: ${DB_PORT:-5432}
       DB_USER: ${DB_USER:-postgres}
-      DB_NAME: ${DB_NAME:-dev-directory}
```

Comment:
this environment variable was overriding the one set by `.env.test`, hence the removal